kingpin/bin/deploy.py

```diff
@@ -120,7 +120,7 @@ def main():
     if args.build_only:
         try:
             get_main_actor(dry=False)
-        except actor_exceptions.ActorException as e:
+        except Exception as e:
```

Comment:
That was an intentional decision by you and I back in the day.. the idea was that any uncaught exception _DID_ crash the app on purpose because its not expected.